docs-next/pages/apis/schema.mdx

```diff
@@ -2,10 +2,219 @@ import { Markdown } from '../../components/Page';
 
 # Schema API
 
+The `lists` property of the [system configuration](./config) object is where you define the data model, or schema, of your Keystone system.
+To setup the `lists` property of the system configuration you need to use the `createSchema()` function.
+This function accepts an object with list names as keys, and `list()` configurations as values.
+
+```typescript
+import { config, createSchema, list } from '@keystone-next/keystone/schema';
+
+export default config({
+  lists: createSchema({
+    ListName: list({
+      fields: { ... },
+      idField: { ... },
+      access: { ... },
+      ui: { ... },
+      hooks: { ... },
+      graphql: { ... },
+      description: '...',
+    }),
+    ...
+  }),
+  ...
+});
 ```
-import type { ListSchemaConfig } from '@keystone-next/types';
+
+This document will explain the configuration options which can be used with the `list()` function.
+
+## `fields`
+
+The `fields` option defines the names, types, and configuration of the fields in the list.
+The configuration option takes an object with field names as keys, and configured field types as values.
```

Comment:
which configuration option are you referring to here? are you just referring to the `fields` option here? 

if yes 
```suggestion
This configuration option takes an object with field names as keys, and configured field types as values.
```